src/data-fetcher-loop/data-fetcher-loop.ts

```diff
@@ -56,7 +56,7 @@ export const callSignedApi = async (url: string, timeout: number): Promise<Signe
 };
 
 export const runDataFetcher = async () => {
-  return logger.runWithContext({ dataFetcherCoordinatorId: Date.now().toString() }, async () => {
+  return logger.runWithContext({ dataFetcherCoordinatorId: new Date().toISOString() }, async () => {
```

Comment:
If we continue with time, I'd prefer timestamp rather than ISO string cuz it's easier to search logs with timestamp. When you double-click on timestamp it will select all, it's not possible with ISO format 😄

Reply:
:D UUID would have the same issue though.

Anyway, let's close this and stick with timestamp.

Reply:
Yeah, but the JS timestamp is in MS so that's very low probability that the loops start at the same time. Anyway, 32-bit hex from node:crypto + a timestamp logged at the start of the coordinator seems like a trade-off that everyone is going to be happy with. WDYT?

Reply:
Yes, it looks like sweet-spot.